CHANGELOG_PENDING.md

```diff
@@ -25,3 +25,4 @@ IMPROVEMENTS:
 
 BUG FIXES:
 - [node] \#2294 Delay starting node until Genesis time
+- [autofile] \#2428 Group.RotateFile need call Flush() before rename.
```

Comment:
Could you please add your github username in the end? `before rename (@goolAdapter)`